src/Microsoft.ML.PipelineInference/Microsoft.ML.PipelineInference.csproj

```diff
@@ -15,9 +15,9 @@
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.ML.Api\Microsoft.ML.Api.csproj" />
     <ProjectReference Include="..\Microsoft.ML.Core\Microsoft.ML.Core.csproj" />
+    <ProjectReference Include="..\Microsoft.ML.Legacy\Microsoft.ML.Legacy.csproj" />
```

Comment:
Hi @justinormont , shortly we will be removing this dependency, since we are preparing to delete that project. Pursuant to #689, any code that cannot be easily rewritten to adapt itself to that will have to be removed.